source/Octopus.Tentacle.Tests/Octopus.Tentacle.Tests.csproj

```diff
@@ -25,7 +25,6 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Octopus.Tentacle\Octopus.Tentacle.csproj" />
-    <PackageReference Include="Halibut" Version="4.4.9" />
```

Comment:
I think this reference doesn't need to exist since it would be included via `Octopus.Tentacle.csproj`